examples/custom-hooks/package.json

```diff
@@ -8,7 +8,8 @@
 	},
 	"scripts": {
 		"fuzz": "jazzer fuzz -i jpeg-js -e nothing -h custom-hooks --sync",
-		"dryRun": "jazzer fuzz -i jpeg-js -e nothing --sync -h custom-hooks -- -runs=100 -seed=123456789"
+		"dryRun": "jazzer fuzz -i jpeg-js -e nothing --sync -h custom-hooks -- -runs=100 -seed=123456789",
+		"coverage": "jazzer fuzz -i jpeg-js -i fuzz.js -i custom-hooks.js -e nothing -h custom-hooks --sync --corpus corpus --coverage -- -max_total_time=2"
```

Comment:
Do we need all these items in the corpus? We don't seem to test the exact coverage and I think it contains too many seeds with long names

Reply:
It was initially with --coverage and -runs=0, where an empty corpus was bad for demonstration purposes.
But now with -max_total_time=2 we don't need the corpus anymore.
I will remove it!